cms/templates/js/active-video-upload.underscore

```diff
@@ -1,3 +1,8 @@
 <h4 class="video-detail-name"><%- fileName %></h4>
 <progress class="video-detail-progress" value="<%= progress %>"></progress>
-<p class="video-detail-status"><%- gettext(status) %></p>
+<p class="video-detail-status">
+  <%- gettext(
```

Comment:
Here, this is a varible name, not a string literal that need to be extracted. But the library does not distinguish them and it will extract anything that looks like a string at the position of first argument. So separate them into two lines will make the the varible name 'status' not be mistakenly extracted.